scenarios/kubernetes_janitor.py

```diff
@@ -94,6 +94,8 @@ def clean_ssh_keys(project):
     'k8s-jkns-pr-kubeadm': 3,
     'k8s-jkns-pr-kubemark': 3,
     'k8s-jkns-pr-node-e2e': 3,
+    'k8s-jkns-e2e-gke-gpus': 3,
+    'k8s-jkns-pr-gce-gpus': 3,
 }
```

Comment:
nite: can we add a comment explaining this dict here?

Reply:
added comments about the magic number 3